docs/REST-API/18-PCL-Overview.md

```diff
@@ -0,0 +1,590 @@
+- other operations: `==`, `in`, `exists`, `now`
+
+The operations described below will have an initial codeblock which shows the syntax of the operation. It will look something like:
+
+```
+[integer] < [integer] -> [boolean]
+```
+
+where to the left of the `->` we show the types of the arguments and the way the operation is used, and to the right of the `->`, we show the type of the return value. In both cases the type will be surrounded by `[]` to distinguish it from the operation name.
+
+#### Boolean Operations
+
+##### `not`
+
+Returns the boolean `not` of the argument, i.e. the opposite boolean value to the one passed in as an argument.
+
+```
+not [boolean] -> [boolean]
+```
+
+Should be used in combination with the other operators.  Has higher precedence than the other boolean operators - use parentheses if you need to change the order of precdence.
+
+```
+{
+  "data": {
+    "foo: "code"
+  }
+}
+
+not data.foo matches 'www' -> true
+not (data.foo exists and data.foo matches 'code') -> false
+```
+
+A `not` will also return the opposite of [Booleans with Warnings][Boolean Evaluation With Warnings] and treats [Non-Boolean Evaluations][Non-Boolean Evaluation] as `false`.
+
+```
+data.invalid_path == 'staging' -> false (Type mismatch: `==` requires the same type on both sides but got `[nil] == [string]`)
+not data.invalid_path == 'staging' -> true (Type mismatch: `==` requires the same type on both sides but got `[nil] == [string]`)
+
+1 > 'string' -> false (Type mismatch: `>` can only compare two [number] or [date] types but got `[number] > [string]`)
+not 1 > 'string' -> true (Type mismatch: `>` can only compare two [number] or [date] types but got `[number] > [string]`)
+```
+
+##### `and`
+
+Returns the boolean `and` of the two arguments, i.e. returns `true` if both arguments are `true` and `false` otherwise.
+
+```
+[boolean] and [boolean] -> [boolean]
+```
+
+No conversions are performed; returns `false` if either arg is not a `boolean` type. Returns `false` if either args is a [Non-Boolean Evaluation]. Can be used with boolean literals or be used for creating nested expressions by joining the other operators. Has higher precedence than `or`, but lower predence than `not`.
+
+##### `or`
+
+Returns the boolean `or` of the two arguments, i.e. returns `false` if both arguments are `false` and `true` otherwise.
+
+```
+[boolean] or [boolean] -> [boolean]
+```
+
+No conversions are performed; returns `false` if either arg is not a `boolean`. Treats [Non-Boolean Evaluation] as `false`. Can be used with boolean literals or be used for creating nested expressions by joining the other operators.  Has lower precedence than `and`.
+
+#### Numeric Comparison Operations
+
+##### `>`, `>=`, `<`, `<=`
+
+```
+[datetime] >  [datetime] -> [boolean]
+[datetime] >= [datetime] -> [boolean]
+[datetime] <  [datetime] -> [boolean]
+[datetime] <= [datetime] -> [boolean]
```

Comment:
I consider it a bug that the user can only use now on the left, so I didn't want the documentation to reflect that. It's in the post-GA epic, so I figured it would be fixed soon enough to match the documentation: https://pagerduty.atlassian.net/browse/ORCA-3115